packages/jest-jasmine2/src/jasmineAsyncInstall.js

```diff
@@ -16,7 +16,7 @@ import type {Global} from 'types/Global';
 
 import isGeneratorFn from 'is-generator-fn';
 import co from 'co';
-import checkIsError from './is_error';
+import checkIsError from './isError';
```

Comment:
as mentioned in https://github.com/facebook/jest/issues/4969#issuecomment-433683303 we can change the variable name as well